dev/com.ibm.ws.request.timing/resources/com/ibm/ws/request/timing/internal/resources/LoggingMessages.nlsprops

```diff
@@ -27,6 +27,11 @@ REQUEST_TIMER_WARNING=TRAS0112W: Request {0} has been running on thread {1} for
 REQUEST_TIMER_WARNING.explanation=The request has been running for longer than the configured slow request duration. The information in the table shows what events have already run as part of the request. Events that are still running are indicated with a + next to the duration. The default value for slow request duration is 10 seconds, check the server.xml file for the current value.
 REQUEST_TIMER_WARNING.useraction=Use the information in the table to determine what part of the request is slower than expected.  If many warnings are provided for different requests at nearly the same time it may indicate that something slowed down the entire server process, such as another process on the same system consuming a large amount of processing resources.
 
+# Information message telling the user that the previously detected slow request has finished in a set amount of time.
+REQUEST_TIMER_FINISH_SLOW=TRAS0113I: The {0} request on the {1} thread, which was previously detected as slow, completed after {2} ms.
```

Comment:
For consistency with TRAS0112W it would be better to say...

TRAS0113I: Request {0} running on thread {1}, which was previously detected as slow, has completed after {2}ms.

Reply:
@Charlotte-Holt  ^^^.  Wanted to make sure you saw this one.  My pref would be to be consistent with the message that initially says the request is slow.  Is that ok, or are we better to try to clean up on the new message even if it's less consistent that way?

Reply:
Hi, @donbourne. From an ID perspective, we try to clean up the messages as we review them, so I'd ask that we go with my suggestions unless you think it's absolutely necessary to maintain consistency here.